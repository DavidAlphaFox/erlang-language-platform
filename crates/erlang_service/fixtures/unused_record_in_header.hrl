-record(my_record, {my_field}).
