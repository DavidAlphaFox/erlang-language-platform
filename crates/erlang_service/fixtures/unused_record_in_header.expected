AST
{attribute,{0,0},file,{"fixtures/unused_record_in_header.hrl",0}}.
{attribute,{0,30},
           record,
           {my_record,[{record_field,{20,28},{atom,{20,28},my_field}}]}}.
{eof,{32,32}}.


STUB
{attribute,{0,0},file,{"fixtures/unused_record_in_header.hrl",0}}.
{attribute,{0,30},
           record,
           {my_record,[{record_field,{20,28},{atom,{20,28},my_field}}]}}.


WARNINGS
[]

ERRORS
[]
